internal/kubernetes/deployment_test.go

```diff
@@ -99,12 +99,15 @@ var _ = Describe("Deployment", func() {
 					{
 						Type:   "progressing",
 						Reason: "progressdeadlineexceeded",
+						// Where this error is defined in k8s: https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/kubectl/pkg/polymorphichelpers/rollout_status.go#L78
+						Message: "deployment exceeded its progress deadline",
```

Comment:
I put the link in the comment so we know where to go if the message format ever changes.